packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -293,6 +293,28 @@ class TransactionService {
             if (fid > 0) {
               tr.experience.fid = fid
             }
+
+            let longtaskCount = 0
+            let longtaskDuration = 0
+            let longtaskMax = 0
+            for (let i = 0; i < tr.spans.length; i++) {
```

Comment:
Even though this works, The approach doesn't feel right to me personally as we are looping over 100's of spans on an average page even for browsers that does not support long tasks.